var/spack/repos/builtin/packages/py-nglview/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyNglview(PythonPackage):
+    """Jupyter widget to interactively view molecular structures and trajectories."""
+
+    homepage = "http://nglviewer.org"
+    pypi = "nglview/nglview-3.0.8.tar.gz"
+
+    maintainers("w8jcik")
+
+    version("3.0.8", sha256="f9e468cd813dac319cbeca6ae20ae099008ff3a06399f5d23d75582dde28623a")
+
+    depends_on("py-ipywidgets@7:", type="run")
+    depends_on("py-jupyterlab-widgets", type="run")
+    depends_on("py-numpy", type="run")
```

Comment:
Should be `type=("build", "run")` in order to get `spack install --test=root` to pass.